security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,77 @@
+
+[id="compliance-operator-release-notes-0-1-44"]
+== Release Notes for Compliance Operator 0.1.44
+Issued: 2021-
+
+The Compliance Operator 0.1.44 is now available with enhancements and bug fixes.
+
+[id="compliance-operator-0-1-44-new-features-and-enhancements"]
+=== New features and enhancements
+
+* To control whether scan scheduling is strict or not strict - a new `strictNode` option is available for `ComplianceScan`, `ComplianceSuite` and `ScanSetting` scans. This option defaults to true (strict), meaning that the operator will error out if a scan pod can't be scheduled. Switching the option to true makes the scan more permissive an go forward. Useful for clouds with ephemeral nodes.
++
+* In this release, the `nodeSelector` and `tolerations` in the `ScanSettings` object are now configurable  This enables deployers to configure where the Result Server will run, and thus what node will host the persistent volume that will contain the raw results. This allows us to schedule a pod that makes use of a persistent volume on the master nodes.
++
+* In this release, the Compliance Operator can now remediate `KubeletConfig` objects.
++
+* In this release, an enhancement had been added for content developers. An error is added to a comment for result objects that differentiates between objects that don't exist in the cluster versus objects that can't be fetched.
++
+* In this release, rule objects now contain two new attributes, `checkType` and `description`. These attributes allow the user to see if the rule pertains to a node check or to a platform check and also allows the user to audit what the rule check does.
++
+* In this release, the remediation template now allows multi-value variables. 
++
+* With this update, the Compliance Operator can change remediations based on variables that are set in the compliance profile. This is useful for remediations that include deployment-specific values such as time outs, NTP server host names, or similar. Additionally, the `ComplianceCheckResult` objects now use the label `compliance.openshift.io/check-has-value` that lists the variables a check can use.
++
+* This enhancement removes the requirement to extend an existing Profile in order to create a Tailored Profile. The `extends` field in the `TailoredProfile` CRD is no longer mandatory. You can now select a list of rule objects to create a tailor profile from scratch. Note: You will need to select whether your profile applies to nodes or the platform by setting the `compliance.openshift.io/product-type:` annotation or by setting the `-node` suffix for the `TailoredProfile` CR.
++
+* Previously, only master node taints were tolerated. Now, the Compliance Operator tolerates all taints, making it possible to schedule scans on all nodes.
```

Comment:
Feature: The scan pods now tolerate all node taints

Reason: Previously, the scan pods would only be able to tolerate the `node-role.kubernetes.io/master` taint, meaning they would either run on nodes with no taints or on nodes with the `node-role.kubernetes.io/master` taint. However, some deployments use custom taints for their nodes, which would result in the scans not being scheduled on those nodes with custom taints.

Result: The detault tolerations were changed to "tolerate everything". Now, the CO is able to schedule scans on all nodes irrespective of the taints they have.